app/src/main/java/de/danoeh/antennapod/fragment/ItemPagerFragment.java

```diff
@@ -76,31 +77,21 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         feedItems = getArguments().getLongArray(ARG_FEEDITEMS);
         int feedItemPos = getArguments().getInt(ARG_FEEDITEM_POS);
 
-        ViewPager pager = layout.findViewById(R.id.pager);
+        ViewPager2 pager = layout.findViewById(R.id.pager);
         // FragmentStatePagerAdapter documentation:
         // > When using FragmentStatePagerAdapter the host ViewPager must have a valid ID set.
         // When opening multiple ItemPagerFragments by clicking "item" -> "visit podcast" -> "item" -> etc,
         // the ID is no longer unique and FragmentStatePagerAdapter does not display any pages.
         int newId = ViewCompat.generateViewId();
         pager.setId(newId);
-        pager.setAdapter(new ItemPagerAdapter());
+        pager.setAdapter(new ItemPagerAdapter(this));
         pager.setCurrentItem(feedItemPos);
```

Comment:
When clicking an episode (not the first one in the list), I now see an animation scrolling through the pages of other episodes. Not sure what to do about that.

Reply:
Done https://github.com/AntennaPod/AntennaPod/pull/4020/commits/e2ff176455db0f8fc43e0c1e8d92cdf80a316881